src/routes/index.ts

```diff
@@ -37,8 +38,7 @@ router.get("/knex/users", (req, res, next) => {
 /**
  * Kruttika ROUTES
  */
-router.get("/kru", (req, res) => {
-  res.send("Kruttika's point!")
-})
+//router.post("/auth/login", AuthController.Login)
+router.post("/addcompany", CompanyController.AddCompany)
```

Comment:
Don't make route names like this ('/addCompany). This is what the GET, POST, UPDATE & DELETE http methods are for. Here we are creating a new company so we just do POST /company.
-  If we wants to get a company's details we'd do GET /company/:company_id
- to update we'd do UPDATE /company/:company_id
- to deleted we'd do DELETE /company/:company_id 
You see how you can hit the same endpoint but with 3 different operations, that do 3 different thing 

```suggestion
router.post("/company", CompanyController.AddCompany)
```